### DISCUSSION TITLE
OrderLine CustomField with Translatable Custom Entity

### DISCUSSION THREAD

**Discussion by cdiban-loads** [vendurehq/vendure#1764] [General]
Hi Everyone!

I'm seeking help with a use case regarding relation type custom fields in order lines but with translations. My entity has translation code and other properties but when I try to view the order detail I have an error that says "Cannot return null for non-nullable field..." I changed my ApiExtension to nullable, its work but I can't get the properties with the data.

Is it possible to fix this implementation? I'm doing something wrong?

![image](https://user-images.githubusercontent.com/108639323/188004005-a063b1de-bd1f-475a-92e1-95082cf408cd.png)

Thanks!

**Comment by michaelbromley** [comment]
Hi,
I think this might be related to #1664. Can you provide a minimal reproduction of the issue?
